code/game/objects/random/rig.dm

```diff
@@ -41,4 +41,99 @@
 		module.lose_modules(cnd)
 		module.misconfigure(cnd)
 		module.sabotage_cell()
-		module.sabotage_tank()
\ No newline at end of file
+		module.sabotage_tank()
+
+
+
+//Installable modules!
+
+/obj/random/rig_module
+
```

Comment:
So does this just not get a name? Or did you forget to put something here?

Reply:
will fix

Reply:
fixed